.razorops.yaml

```diff
@@ -1,58 +1,27 @@
-tasks:
-  build-image:
-    type: build
-    image: iutest/razorops_test
-    push: false
-    tags: ["latest"]
-    dockerfile: .ci/docker/centos/Dockerfile
-    context: .
-    when: &condition >
-      branch in ('master', 'develop') ||
-      branch =~ 'razorops.*'
+version: v2
 
-  test:
-    image: iutest/razorops_test
-    commands:
-      # - cd /iutest/test
+tasks:
+  run-test:
+    variables:
+    - PARALLELS=-j8
+    steps:
+    - checkout
+    - commands:
       - cd test
-      - make
-      - make test
+      - make $PARALLELS && make run_minimum_tests
       - make report
       - make clean
-    when: *condition
-
-  # build-gcc3-image:
-  #   type: build
-  #   image: iutest/razorops_gcc3
-  #   push: false
-  #   tags: ["latest"]
-  #   dockerfile: .ci/docker/centos/gcc3/Dockerfile
-  #   context: .
-  #   when: *condition
-
-  # gcc3-test:
-  #   image: iutest/razorops_gcc3
-  #   commands:
-  #     - cd /iutest/test
-  #     - make
-  #     - make test
-  #     - make clean
-  #   when: *condition
 
-  # centos7:
-  #   image: centos/devtoolset-7-toolchain-centos7:latest
-  #   commands:
-  #     - cd test
-  #     - make clean
-  #     - make
-  #     - make test
-  #     - make clean
-  #   when: *condition
+  lint-editorconfig:
+    steps:
+    - checkout
+    - cache/pull: npm-editor-{{ checksum "package.json" }}
+    - commands:
+      - npm install
+      - npm run lint:editorconfig
+    - cache/push:
+        key: npm-editor-{{ checksum "package.json" }}
+        paths: [node_modules]
 
-  # gcc3-centos5:
-  #   image: srzzumix/gcc3:centos5
-  #   commands:
-  #     - find . -type f \( -name \*.hpp -or -name \*.cpp -or -name \*.ipp \) -exec sed -i -e '1s/^\xEF\xBB\xBF//' {} \;
-  #     - make -C test
-  #     - make -C test test
-  #   when: *condition
+trigger:
+  when: branch in ('master', 'develop') || branch =~ 'razorops.*'
```

Comment:
RazorOps doesn't seem to be running.
The condition of Triggers seems to use if.

https://docs.razorops.com/pipeline/triggers/

Reply:
I guess the syntax is correct, but Razorops doesn't support builds from forked repos atm.